hack/local-up-cluster.sh

```diff
@@ -94,6 +94,13 @@ export KUBE_PANIC_WATCH_DECODE_ERROR
 ADMISSION_CONTROL=${ADMISSION_CONTROL:-""}
```

Comment:
this already supposedly honors an already-set $ADMISSION_CONTROL

we could just do this below and let people completely override:
```
# Default if unset
if [[ -z "$ADMISSION_CONTROL" ]]; then
  ADMISSION_CONTROL=...
fi
```

that seems better than adding another envvar and trying to pick the perfect injection point

Reply:
The problem with that is that its a lot more work. I would now need to see what's there in the script file each time I upgraded my master so I could then populate my env var.

I'm trying to find a way to enable the podPreset admission controller with minimal (zero?) effort upon upgrading. I understand it doesn't work for all cases, because we can't really do that, but for the easy case of "just wanting to add a stand-alone one to the end of the list" this seems like it should work.